index.js

```diff
@@ -750,27 +738,6 @@ class ClientServiceBase {
     return true;
   }
 
-  /**
-   * @param {LedgerClient|AuditorClient} client
-   * @param {LedgerValidationRequest} request
-   * @return {Promise}
-   * @throws {ClientError}
-   */
-  async _validateLedgerAsync(client, request) {
-    const promise = new Promise((resolve, reject) => {
-      client.validateLedger(request, this.metadata, (err, response) => {
-        if (err) {
-          reject(err);
-        } else {
-          resolve(
-              LedgerValidationResult.fromGrpcLedgerValidationResponse(response),
-          );
-        }
-      });
-    });
-    return promise;
-  }
-
```

Comment:
Deleted due to https://github.com/scalar-labs/scalardl-javascript-sdk-base/pull/85/files#r969210304

We don't need it anymore.